### DISCUSSION TITLE
Postal compatibility with Chatwoot continuity

### DISCUSSION THREAD

**Discussion by uhlhosting** [postalserver/postal#2475] [Help with using Postal]
Hi,

I am wondering if there could be a way to use Postal for continuity with Chatwoot: https://www.chatwoot.com/docs/self-hosted/configuration/features/email-channel/conversation-continuity . 

Thinking since postal is similar to other products supported by them, maybe there could be some ways to add support for Postal to it.

If so, what would that path be?

**Comment by willpower232** [comment]
do you mean for Postal to receive an email and pass it through to Chatwoot? it looks like some code would need to be added to Chatwoot for it to parse the incoming message from Postal
